openqabot/loader/config.py

```diff
@@ -1,24 +1,32 @@
 # Copyright SUSE LLC
 # SPDX-License-Identifier: MIT
+from __future__ import annotations
```

Comment:
Note that 3.14 doesnt need this.the import will be redundant and should be removed. Maybe add a comment like for <3.14